README.md

```diff
@@ -53,6 +53,19 @@ and health-checking mechanisms.
   ```
 
 * Install puppet-agent (&ge; 6.13.0)
+  
+  Check facter version details, if already installed / exist in system
+  ```sh
+  facter -v
+  ls -ll /usr/bin/facter
+  ```
+  Supported version is **facter >= 3.14.8**, If the facter version is **< 3.14.8** then follow the steps below: 
+  ```sh
+  yum erase -y $(rpm -q --whatprovides /usr/bin/facter) || rm -fv /usr/bin/facter
+  yum list installed | grep "puppet-agent"
+  yum remove puppet-agent.x86_64
```

Comment:
These two lines are not needed (already covered by line 64)

Reply:
Done